.pre-commit-config.yaml

```diff
@@ -9,15 +9,53 @@ repos:
       - id: check-xml
       - id: check-yaml
       - id: end-of-file-fixer
+        exclude: courts_schema_types_autogenerated\.py
```

Comment:
Can we just exclude this across the board by adding it to the top of the config? Seems daft to add it in all the stages when we know it's autogenerated and will probably upset a lot of quality tools.